src/password/landing.js

```diff
@@ -1,17 +1,22 @@
 document.addEventListener('DOMContentLoaded', () => {
 if(localStorage.getItem('storedHashedPassword')){
+    document.getElementById('welcome-button').style.display = 'none'
 //Wait for input password to be submitted
 document.getElementById('input-password-form').addEventListener('submit', async function(event){
+    console.log("here")
     event.preventDefault();
     const inputPassword = document.getElementById('password').value;
+    //hashed version of the input
     const hashedInputPassword = await hashPassword(inputPassword);
+    //retrive the stored hashed password
     const storedHashedPassword = localStorage.getItem('storedHashedPassword');
```

Comment:
We may want to add a new api file in the backend-storage folder for this, then import the getter and setter functions from it. 

This would help be consistent with the rest of our code and keep it decoupled from localStorage so that in the future, it is possible to easily change the backend storage just by changing the function bodies inside the backend-storage folder.